### DISCUSSION TITLE
Did form.Provider get removed?

### DISCUSSION THREAD

**Discussion by noahkurz** [TanStack/form#773] [Q&A] [answered]
Let me start by saying I love this library. I think it is by far the best form library in terms of DevEx. I was working with it on a new project this morning and noticed <form.Provider> got removed. This is essential to use this library in a react native application. Is there any way we could get that added back in?

**Comment by crutchcorn** [comment, marked as answer]
It was removed but is not needed for React Native support in any way. Form should continue to work as well (if not better) in RN today as it did then.

**Reply by noahkurz** [reply]
Im currently getting this error, following the docs, on the latest version. 

Invariant Violation: View config getter callback for component `form` must be a function (received `undefined`). Make sure to start component names with a capital letter.

When I downgrade and use the provider the error resolves. Can you think of anything that would need to be different for RN in terms of config?

**Reply by crutchcorn** [reply]
This is quite strange.... Can you share a minimal reproduction repo with me in a new GH issue so I can investigate? I'm not understanding what the error you're seeing is saying without the code

**Reply by noahkurz** [reply]
Yes, of course. I'll do that right away. I appreciate the help!

**Reply by noahkurz** [reply]
Added under [774](https://github.com/TanStack/form/issues/774)
